extensions/binding/org.eclipse.smarthome.binding.ntp.test/src/test/java/org/eclipse/smarthome/binding/ntp/test/NtpOSGiTest.java

```diff
@@ -267,8 +264,7 @@ public void testStringChannelDefaultTimeZoneUpdate() {
 
         String timeZoneFromItemRegistry = getStringChannelTimeZoneFromItemRegistry();
 
-        assertThat("The string channel was not updated with the right timezone", timeZoneFromItemRegistry,
-                is(anyOf(equalTo(expectedTimeZoneEEST), equalTo(expectedTimeZoneEET))));
+        assertThat(timeZoneFromItemRegistry, is(anyOf(equalTo(expectedTimeZoneEEST), equalTo(expectedTimeZoneEET))));
```

Comment:
same as above: Is EEST and EET different and why is this asserted the one _or_ the other?